scripts/qiita

```diff
@@ -130,24 +130,31 @@ def load_study(owner, title, info):
               help="The parent artifacts from which the new artifact has been "
                    "processed",
               type=click.IntRange(1))
-@click.option('--command', required=False,
-              help="The command used to process the new artifact",
-              type=click.IntRange(1))
-@click.option('--parameter_set', required=False,
-              help="The parameter set used to process the new artifact",
+@click.option('--dflt_params', required=False,
```

Comment:
I think a new command is the way to go, as @antgonza points out. Is it possible to open an issue? I think it is a nice to have, but not necessarily a blocker right now, and it will take more time than the one that I've right now.